### PR TITLE
[OP-50740] Set width of primer form inputs to :large in storage forms

### PR DESCRIPTION
**Pull request by apfohl**
This PR introduces the new `input_width` attribute for `select` and `text_field` primer inputs and change the visual appearance of the inputs for the storage administration form for editing and creating storage base information. They are set to `:large`.

Screenshot 2023-11-21 at 11 54 46 AM

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -97,7 +97,7 @@
     "@ngneat/content-loader": "^7.0.0",
     "@ngx-formly/core": "^6.1.4",
     "@openproject/octicons-angular": "^19.7.0",
-    "@openproject/primer-view-components": "^0.13.1",
+    "@openproject/primer-view-components": "^0.16.0",
```
Comment (HDinger) [inline frontend/package.json:100]: Please also change the version in the [`overrides` section](https://github.com/opf/openproject/blob/dev/frontend/package.json#L178) at the very bottom.

File: modules/storages/app/forms/storages/admin/provider_type_select_form.rb [hunk 2]
```diff
@@ -61,7 +61,8 @@ def default_select_list_options(storage)
         caption: nil,
         include_blank: false,
         required: true,
-        disabled: storage.persisted?
+        disabled: storage.persisted?,
+        input_width: :large
```
Comment (HDinger) [inline modules/storages/app/forms/storages/admin/provider_type_select_form.rb]: Taken from the figma design, this should be smaller than the rest. So maybe `:medium` would be better here?

**Review by HDinger — COMMENTED** [review]
Hi @apfohl 

I like that, it looks so much better than before! 🥳 I just have two minor remarks, after that it is good to merge.

**Review by akabiru — COMMENTED** [review]
Nice one 👍🏾  Ditto on input sizes- the select field should be `:small` I think

- [x] Forgot to update https://github.com/opf/openproject/blob/dev/modules/storages/app/forms/storages/admin/oauth_client_form.rb

🚧 P.S. You can ignore the ClipboardCopy component as it's different and @HDinger has offered to take it up in https://community.openproject.org/wp/50741

File: modules/storages/app/forms/storages/admin/provider_type_select_form.rb (hunk 2 above)
Comment (akabiru) [reply modules/storages/app/forms/storages/admin/provider_type_select_form.rb]: updated to `:small` 👍🏾

File: frontend/package-lock.json [hunk 3]
```diff
@@ -4493,9 +4493,9 @@
       }
     },
     "node_modules/@oddbird/popover-polyfill": {
-      "version": "0.2.3",
-      "resolved": "https://registry.npmjs.org/@oddbird/popover-polyfill/-/popover-polyfill-0.2.3.tgz",
-      "integrity": "sha512-XDK+V/gUeE4NEsWp79eVzhlK3wuVcRDJuaas63qo0IJLJpyOLHqycJLFYvuq8kebgT1nl87P3sbSb5ZN6Vyf5g=="
+      "version": "0.3.2",
```
Comment (akabiru) [inline frontend/package-lock.json:4496]: @HDinger are these updates OK?

Comment (HDinger) [reply frontend/package-lock.json:4496]: Yes. Primer has updated its depedencies for `popover-polyfill` so we have to update as well

**Review by HDinger — APPROVED** [review]

**Pull request merged**